packages/forms/src/UIForm/fieldsets/Array/DefaultArrayTemplate.component.test.js

```diff
@@ -107,4 +107,25 @@ describe('Default Array Template component', () => {
 		// then
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
+
+	it('should render with error', () => {
+		// when
+		const wrapper = shallow(
+			<DefaultArrayTemplate.WrappedComponent
+				canReorder
+				id={'my-template'}
+				onAdd={jest.fn()}
+				onRemove={jest.fn()}
+				onReorder={jest.fn()}
+				renderItem={index => <div>Render item {index}</div>}
+				schema={schema}
+				value={value}
+				errorMessage={'This is an error'}
+				isValid={false}
+			/>,
+		);
+
+		// then
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
Prefer expecting specific things (like having an errorId and descriptionId in the Message rendered) than a snapshot.
To avoid having to update n unrelated snapshots for just a minor change. And also not knowing what to look at in the snapshot to check if the expectation is fulfilled.